.github/workflows/functional-test-cloud.yaml

```diff
@@ -758,6 +758,12 @@ jobs:
             }
           }
           EOF
+      - name: Restore Bicep artifacts before running functional tests
+        run: |
+          # Restore Radius Bicep types
+          bicep build ./test/functional-portable/corerp/cloud/resources/testdata/corerp-azure-connection-database-service.bicep
+          # Restore AWS Bicep types 
+          bicep build ./test/functional-portable/corerp/cloud/resources/testdata/aws-s3-bucket.bicep
```

Comment:
Do the tests need the files generated by these two commands to be able to run successfully? If so, should we add those files to the repository itself?

I think that to be able to run these tests locally, this is also something we need to run. Is that right?

Reply:
No, they're not needed. They're an output of the build command but we're more interested in the artifacts being restored in this step. 

Yes to the second question. The `bicep build` command gets run when we deploy bicep templates. `bicep build` automatically restores the artifacts which might be why we've been seeing flaky restoration failures. This change is added so the bicep artifacts are restored before the tests start and `bicep build` is run